sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 - Changed return type of `BlobClient::GetProperties` to `BlobProperties`.
 - Changed return type of `BlobClinet::GetTags` to `std::map<std::string, std::string>`.
 - Removed `PreviousShareSnapshot` from `GetShareFileRangeListOptions`, use `ShareFileClient::GetRangeListDiff` instead.
+- Removed `Blob` from the name of lease options and return types, like `AcquireBlobLeaseOptions` was renamed to `AcquireLeaseOptions`.
+- Removed `Blob` prefix from the name of structs `BlobGeoReplication`, `BlobLeaseStatus`, `BlobLeaseState`, `BlobLeaseDurationType`, `BlobAnalyticsLogging`, `BlobMetrics`, `BlobCorsRule`, `BlobRetentionPolicy`, `BlobStaticWebsite`, `BlobArchiveStatus` and `BlobGeoReplictionStatus`.
```

Comment:
None of these structs are returned directly form any client API, correct (as in, they are always a field in some other struct that gets returned)?

Reply:
Yep, but it's still a breaking change

Reply:
Right, that's fine. Just understanding the scope of the naming change.